addon/hifi-connections/dummy-connection.js

```diff
@@ -8,67 +8,89 @@ let ClassMethods = Ember.Mixin.create({
   canPlayMimeType: () => true,
 });
 
-let DummyConnection = Ember.Object.extend(Ember.Evented, DebugLogging, {
+
+let DummyConnection = BaseSound.extend({
   debugName: 'dummyConnection',
-  position: 0,
-  duration: 10000,
-  
-  init() {
-    this.on('audio-played',    () => {
-      this.set('hasPlayed', true);
-      this.set('isLoading', false);
-      this.set('isPlaying', true);
-      this.set('error', null);
-      
-      // recover lost isLoading update
-      this.notifyPropertyChange('isLoading');
-    });
+  _dummy_position: 0,
+  setup() {
+    let {result} = this.getInfoFromUrl();
+    if (result === 'bad') {
+      Ember.run.next(() => this.trigger('audio-load-error', this));
+    }
+    else {
+      Ember.run.next(() => this.trigger('audio-ready', this));
+    }
+  },
+
+  stopTicking: function() {
+    Ember.run.cancel(this.tick);
+  },
+
+  startTicking: function() {
+    this.tick = Ember.run.later(() => {
+      this._setPosition((this._currentPosition() || 0) + 1000);
+      this.startTicking();
+    }, 1000);
```

Comment:
Nope, not worth it after all. Injecting a service into an object without a container (a connection): no bueno.